ui-app/client/src/components/FormStorageEditor/editors/TextTypeEditor.tsx

```diff
@@ -0,0 +1,491 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import {
+	CustomSchema,
+	FormCompDefinition,
+	FormCompValidation,
+	compValidationMap,
+	CompValidations,
+} from '../components/formCommons';
+import { duplicate } from '@fincity/kirun-js';
+import { Dropdown, DropdownOptions } from '../components/Dropdown';
+import CommonCheckBox from '../components/CommonCheckBox';
+import CommonTextBox from '../components/CommonTextBox';
+
+const inputTypeOption: DropdownOptions = [
```

Comment:
Please use the above mentioned, same naming convention for the const outside the component function everywhere.